### PR TITLE
Day7 asserts

### PR DESCRIPTION
**Pull request by J2mshed**

### CODE REVIEW

File: src/test/java/StringExamples.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StringExamples {
+    static String test;
+
+    @Test
+    public static void main(String[] args) {
```
Comment (ydanylchenko) [inline src/test/java/StringExamples.java:8]: It's not valid method signature

File: src/test/java/StringExamples.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StringExamples {
```
Comment (ydanylchenko) [inline src/test/java/StringExamples.java:4]: It's not valid test class name

**Comment by gogr** [comment]
@J2mshed  
I do see you are using the methods, but not verifying. Please update them with asserts

File: src/test/java/StringExamples.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        // uppercase, lowercase
+        System.out.println("AbCdEf".toLowerCase()); // abcdef
+        System.out.println("AbCdEf".toUpperCase()); // ABCDEF
+
+        // equality
+        System.out.println("ABC".equals("abc")); // false
+        System.out.println("abc".equals("abc")); // true
+        System.out.println("abc".equalsIgnoreCase("ABC")); //true
+        System.out.println("abcDD".equalsIgnoreCase("ABCdd")); // true
+        System.out.println("abcDD12ssae22".equalsIgnoreCase("ABCdd12ssae22"));
```
Comment (ydanylchenko) [inline src/test/java/StringExamples.java:56]: Update these ones to asserts instead of sout

File: src/test/java/StringExamples.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+        Assert.assertEquals("assert equal","assert"+" equal");
+        Assert.assertNotEquals("assert equal","assert"+"equal");
+        Assert.assertEquals("ABCD", "abcd".toUpperCase());
+        Assert.assertEquals("abcd", "AbCd".toLowerCase());
+        Assert.assertEquals("assert", ("asse".concat("rt")));
+        Assert.assertEquals("def", "abcdef".substring(3));
+        Assert.assertEquals("de", "abcdef".substring(3,5));
+        Assert.assertEquals(true, "qwerty".startsWith("qw"));
+        Assert.assertEquals(true, "qwerty".endsWith("ty"));
```
Comment (ydanylchenko) [inline src/test/java/StringExamples.java:69]: as per the HW it should be one test per string method, i.e.  
Assert.assertEquals("assert equal","assert"+" equal");
        Assert.assertNotEquals("assert equal","assert"+"equal");
should be in one Test
        Assert.assertEquals("ABCD", "abcd".toUpperCase());
should be in another test
etc.

**Pull request merged**